### ISSUE TITLE
Tab out of quotations, brackets, parentheses, etc...

### ISSUE THREAD

**Issue by DzhangCHES** [a85/POSTMan-Chrome-Extension#11164]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Feature is not related to a problem, just a QoL improvement.

### Describe the solution you'd like

I would like the ability to tab out of quotations, brackets, and parentheses. i.e. hitting tab at the end of a closing quotation, bracket, or parenthesis will move the cursor to the right of that character instead of inserting spaces.

Basically, the TabOut extension for Visual Studio Code by Albert Romkes.

Pretty please, thanks!

### Describe alternatives you've considered

I've disabled auto closing quotations, but I actually like the auto-close, just that the right arrow key is just a little too inconvenient to hit compared to tab.
